src/Sulu/Bundle/AdminBundle/Admin/View/ToolbarAction.php

```diff
@@ -35,11 +35,17 @@ public function __construct(string $type, array $options = [])
         $this->options = $options;
     }
 
+    /**
+     * @return string
+     */
     public function getType()
     {
         return $this->type;
     }
 
+    /**
+     * @return array
```

Comment:
atleast newer version of phpstan forces generic definitions

```suggestion
     * @return array<string, mixed>
```